public/src/components/channelManagement/bannerTests/bannerTestVariantEditor.tsx

```diff
@@ -356,6 +357,8 @@ const BannerTestVariantEditor: React.FC<BannerTestVariantEditorProps> = ({
   const classes = useStyles();
   const setValidationStatusForField = useValidation(onValidationChange);
 
+  const allowVariantTicker = false; // TODO - set based on template
```

Comment:
So until this line is changed, this PR won't have a visible affect on any RRCP pages?

Reply:
that's right